pkgs/top-level/python-packages.nix

```
@@ -2694,6 +2694,12 @@ in {
 
   ffmpeg-python = callPackage ../development/python-modules/ffmpeg-python { };
 
+  fenics = disabledIf isPy27 (callPackage ../development/libraries/science/math/fenics {
+    inherit (pkgs) pkgconfig;
```

preference to use canonical name
```suggestion
    inherit (pkgs) pkg-config;
```